simple_salesforce/bulk.py

```diff
@@ -90,7 +90,7 @@ def _create_job(self, operation, object_name, use_serial,
             use_serial = 0
```

Comment:
Do you guys know what is the reason for this? According to the docs it is expecting `Parallel` or `Serial`

Reply:
It also allows for 0/1 in the concurrency mode header but its just not stated in the documents. If you pass 0 in concurrency mode then it will run the batches in parallel if you pass 1 then it will run them in serial.

Reply:
Thanks for the clarification @jon-wobken